templates/docs/examples/patterns/navigation/_script.js

```diff
@@ -37,8 +37,12 @@ function initNavDropdowns(containerClass) {
     toggle.addEventListener('click', function (e) {
       e.preventDefault();
 
+      const wasOpen = toggle.parentNode.classList.contains('is-active');
       closeAllDropdowns(toggles);
       toggleDropdown(toggle, true);
+      if (!wasOpen) {
+        toggleDropdown(toggle, true);
```

Comment:
I guess you don't have to `toggleDropdown` twice? If it was open, just don't toggle it at all.